test/spoom/context/sorbet_test.rb

```diff
@@ -44,7 +44,7 @@ def test_context_run_srb_from_bundle
 
         context.write_sorbet_config!(".")
         res = context.srb("tc")
-        assert_equal(<<~ERR, res.err)
+        assert_includes(<<~ERR, res.err)
```

Comment:
We're getting warnings that Sorbet needs to add `bigdecimal` in its gemspec because it'll stop being bundled in 3.4.0 https://github.com/ruby/ruby/commit/1c93288f8bbf667cb95eb7137b2fe64213894b77. Which clutters the stderr.

I can't reproduce the warning on a typecheck of the same file locally. So for now I didn't open a Sorbet PR for this.